Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "RxGCXMulticastDNSKit",
+    platforms: [
+        .iOS(.v9),
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "RxGCXMulticastDNSKit",
+            targets: ["RxGCXMulticastDNSKit"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ReactiveX/RxSwift.git", .upToNextMinor(from: "5.0.1")),
+        .package(url: "https://github.com/grandcentrix/GCXMulticastDNSKit.git", .upToNextMinor(from: "1.3.0"))
```

Comment:
I think we should link to `GCXMulticastDNSKit` beginning with a version that supports SPM. 
`1.3.0` does not support that so my proposal would be to create a `1.3.1` and then point to that as min dependency.

Reply:
You're right